AirCasting/ABConnector/BluetoothManager.swift

```diff
@@ -102,6 +102,7 @@ extension BluetoothManager: CBCentralManagerDelegate {
     func centralManager(_ central: CBCentralManager, didDiscover peripheral: CBPeripheral, advertisementData: [String : Any], rssi RSSI: NSNumber) {
         if !devices.contains(peripheral) {
             if peripheral.name != nil {
+                // check if ab is not recording stand alone session at the moment
                 devices.append(peripheral)
             }
         }
```

Comment:
I know it's not yours but could you please change it to a one-liner if statement, please?